forge-gui/res/cardsfolder/m/make_an_example.txt

```diff
@@ -5,5 +5,5 @@ A:SP$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBTwoPiles | SubA
 SVar:DBTwoPiles:DB$ TwoPiles | Defined$ Remembered | Separator$ Remembered | Chooser$ You | ChosenPile$ DBImprint | ValidCards$ Creature.RememberedPlayerCtrl | Zone$ Battlefield
 SVar:DBImprint:DB$ Pump | ImprintCards$ Remembered
 SVar:DBSac:DB$ SacrificeAll | Defined$ Imprinted | SubAbility$ DBCleanup | StackDescription$ Each opponent sacrifices the creatures in their chosen pile. | SpellDescription$ Each opponent sacrifices the creatures in their chosen pile. (Piles can be empty.)
```

Comment:
Same for this one?

Reply:
With this one, I think the remembered would be cleared after the subability resolves before repeating to the next player (with my proposed change in place)

The timing of the sacrifice does seem more important here than the timing of the pump in the other card, so I think we may be stuck with this for now

Hmmm